### ISSUE TITLE
Not all modules are checked out with compatible branches

### ISSUE THREAD

**Issue by jucor** [openeyes/oe_installer#2]
Dear OpenEyes team

I am new to OE, and just installed it for the first time in dev mode, so apologies if my comment is out of place. I have the impression that `install-oe.sh` installs incompatible branches of OpenEyes and of the submodules. 

In particular, [Line 19](https://github.com/openeyes/oe_installer/blob/master/install/install-oe.sh#L19)  checks out the branch `feature/IOLmaster` of OpenEyes, and the branch `master` of [OphInBiometry](https://github.com/openeyes/OphInBiometry).

The problem is that  the `IOLMaster` branch of OpenEyes  [in PatientController.php](https://github.com/openeyes/OpenEyes/blob/feature/IOLMaster/protected/controllers/PatientController.php#L1424) relies on a class `OphInBiometry_Imported_Events`, which is  is not present in the `master` branch of OphInBiometry. In the default current installation provided by oe_installer, this therefore causes a crash of OpenEyes when trying to view a patient, e.g. URL `http://localhost:8888/patient/view/19434`. The logfile complains the autoload part:
`2016/01/21 23:13:57 [error] [php] include(OphInBiometry_Imported_Events.php): failed to open stream: No such file or directory (/var/www/openeyes/protected/yii/framework/YiiBase.php:427)`

The class `OphInBiometry_Imported_Events.php`, however, is present in the branch [https://github.com/openeyes/OphInBiometry/tree/feature/IOLMaster](`feature/IOLMaster`) of repository OphInBiometry.

In conclusion, shouldn't [https://github.com/openeyes/oe_installer/blob/master/install/install-oe.sh#L34](Line 34 of install-oe.sh) request the branch `feature/IOLMaster` for all submodules? You will know that better than me, as I am not sure that branch exists for all modules nor that is meant to be in sync throughout all of them.

But without checking that branch at least for OphInBiometry, the current installation of OpenEyes crashes.

All the best, and thanks for your hard work! I'm happy to help diagnose this further if you think I can be helpful.

**Comment by jucor** [comment]
Tagging @vetusko here since you are the most recent caller of that class from the main OpenEyes repository.

**Comment by biskyt** [comment]
Hi @jucor. Thanks for bringing this to our attention. I have made some changes to the installer that will fix this. Please note that there are a few known issues with using the installer on older code versions. This will mostly be fixed along with the release of version 1.12 in the next couple of weeks,

In the meant-time, the only compatible code version is feature/IOLMaster. Please use `install-oe.sh feature/IOLMaster` during the installer. This should get you a fully working version. I have updated the READ.ME instructions to reflect this. I'll update again once the release/v1.12 release candidate is ready, and again after final release.

**Comment by jucor** [comment]
Thanks @tobyoe. James M told me indeed how focused you all are on the 1.12 release: best of luck to the whole team!
